mapstory/views.py

```diff
@@ -670,7 +666,12 @@ def post(self, *args, **kwargs):
                     "email": confirmation.email_address.email
                 })
             )
-        return redirect(redirect_url)
+        # sign in to the user and redirect them to their profile page
+        username = confirmation.email_address.user.username
+        user = Profile.objects.filter(username__iexact=username).first()
+        user.backend = 'django.contrib.auth.backends.ModelBackend'
```

Comment:
This is a workaround for doing `authenticate` - which I want to do but need the user's password. But with the process of confirming emails there's no way to grab the password that I know of...